chainer/links/loss/hierarchical_softmax.py

```diff
@@ -25,40 +34,74 @@ def get_paths(self):
     def get_codes(self):
         return self.codes
 
+    def get_parent2child(self):
+        return self.parent2child
```

Comment:
Please write brief explanation of this function.
(Comment is OK because this is not a documented API)

Preferably `get_paths` and `get_codes`, too.